// @vuepress/core/.temp/dynamic/vuepress_blog/paginations.js
/**
 * Generated by "@vuepress/plugin-blog"
 */

import sorters from './pageSorters'
import filters from './pageFilters'

export default [{
  pid: "tags",
  id: "嵌入式",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/嵌入式/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "Github",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/Github/","interval":[0,3]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "在线文档",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/在线文档/","interval":[0,2]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "VuePress",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/VuePress/","interval":[0,2]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "持续集成",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/持续集成/","interval":[0,2]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "单片机",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/单片机/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "tags",
  id: "博客",
  filter: filters.tags,
  sorter: sorters.tags,
  pages: [{"path":"/tag/博客/","interval":[0,1]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "categories",
  id: "MCU",
  filter: filters.categories,
  sorter: sorters.categories,
  pages: [{"path":"/categories/MCU/","interval":[0,2]}],
  prevText: "Prev",
  nextText: "Next",
},
{
  pid: "categories",
  id: "前端",
  filter: filters.categories,
  sorter: sorters.categories,
  pages: [{"path":"/categories/前端/","interval":[0,3]}],
  prevText: "Prev",
  nextText: "Next",
}]
